### DISCUSSION TITLE
on PostResource: have ability to create new category (for that post)?

### DISCUSSION THREAD

**Discussion by wivaku** [filamentphp/filament#4947] [Help] [answered]
I have: `Post` `belongsTo` `Category`.

In my `PostResource` I can see/select what category the post belongs to.
This works great for existing categories.

Is it possible -without leaving that Post page-  to quickly add a new category (and connect it to that post)?
I tried using a relation manager, but that does not work for `belongsTo`. It allows me to create new categories, but the category I create is not connected to the post.

**Comment by danharrin** [comment, marked as answer]
https://filamentphp.com/docs/2.x/forms/fields#creating-new-records

**Reply by wivaku** [reply]
Can't believe I missed that one in the documentation, thanks!
